Library/Formula/scalaenv.rb

```diff
@@ -0,0 +1,42 @@
+class Scalaenv < Formula
+	homepage "https://github.com/mazgi/scalaenv"
```

Comment:
Indentation is way off throughout. Looks like your editor is set to use tabs rather than spaces for indentation - It should be a two *space* indentation.